cns/api.go

```diff
@@ -298,8 +298,8 @@ type IpamPoolMonitorStateSnapshot struct {
 
 // Response describes generic response from CNS.
 type Response struct {
-	ReturnCode types.ResponseCode
-	Message    string
+	ReturnCode types.ResponseCode `json:"ReturnCode"`
```

Comment:
not blocking but just fyi just struct tags are superfluous when they are capitalized and match the field name

Reply:
For some reason linter kept flagging it. So added this to satisfy the linter

Reply:
@thatmattlong  @rbtr  I think there is some inconsistency between running linter locally and then pipeline runs. Do you guys see this also?

Reply:
could be a version drift issue

Reply:
@thatmattlong I still prefer to see JSON struct tags, even if they match the field name. It says "I expect to be turning this struct into JSON and I've thought about how clients will consume it."

Reply:
I suspect that matches the spirit of the linter that vipin mentioned